Update readme files (#757)
hyperledger/aries-vcx: aries_vcx/README.md

# aries-vcx

> `aries-vcx` is crate which is rather  "library" than "framework". We strive to be not too 

Reviewer: ```suggestion
`aries-vcx` is more of a "library" rather than "framework". We strive to be not too 
```

opinionated and simply provide building blocks for whatever you want to build. 

You can use `aries-vcx` to build both end-user products (mobile wallets in role of credential 
holder and prover) or server (typically in role of issuer, verifier). 

> `aries-vcx` provides support to:

Reviewer 2: "support for" + ing form (and also better to not start sentence with `aries-vcx` due to capitalization), but I think the original sentence was fine.

Author: reverted

- create encrypted wallet, 
- read/write from/to Indy ledger,
- establish didcomm connections and exchange messages,
- create and process Aries messages to drive Aries protocols.

## `aries-vcx` components

> If you don't want to use `aries-vcx` directly, you might still find useful smaller components it's

Reviewer 2: This is not a very elegant sentence, I would leave at e.g. "The following crates may be consumed independently of `aries-vcx`: ..."

built upon:
- [messages](../messages) - crate for building and parsing Aries messages
- [diddoc](../diddoc) - crate to work with DIDDocs

### Deprecation notice: Message mediation

> Aries-vcx contains built in support for message mediation which is useful for mobile use-cases. However,

Reviewer 2: "built-in", "use cases"

this feature (implemented via `MediatedConnection`) is now deprecated as it's tied to mediator 
implementation [vcxagency-node](https://github.com/AbsaOSS/vcxagencynode) which does not implement

> Aries pick-up protocol.

Reviewer 2: We might add a link to the RFCs here.

# Getting started

> To start aries-vcx in your project, you need to add github dependency to your `Cargo.toml`:

Reviewer 2: "To start using" or "To use"

```toml
aries-vcx = { path = "https://github.com/hyperledger/aries-vcx" }
```

> It's advise to follow through [tutorial](TUTORIAL.md) to check your environment is properly configured.

Reviewer 2: "It's advisable to follow through the following tutorial"

# Projects built with aries-vcx
- [aries-vcx-agent](../agents/rust/aries-vcx-agent) - sample agent with local persistence. Used for cross-compatibility testing with other aries implementations. 
- [unifii_aries_vcx](../uniffi_aries_vcx) - wrapper around `aries-vcx` to generate Swift and Kotlin wrappers for mobile use-cases

> - [libcx](../libvcx) (**deprecated**) - wrapper around `aries-vcx` to build Java, ObjectiveC, NodeJS wrappers. 

Reviewer 2: Objective-C
